pkg/controllers/vdb/verticadb_controller.go

```diff
@@ -231,6 +228,9 @@ func (r *VerticaDBReconciler) constructActors(log logr.Logger, vdb *vapi.Vertica
 		MakeObjReconciler(r, log, vdb, pfacts, ObjReconcileModeAll),
 		// Set version info in the annotations and check that it is the minimum
 		MakeImageVersionReconciler(r, log, vdb, prunner, pfacts, false),
+		// Check NMA running mode and report error when not supported by
+		// the server version
+		MakeNMARunningModeReconciler(r, log, vdb),
```

Comment:
Is there something in particular we want to do in this reconciler? For now I just check the version and the annotations

Reply:
We can probably remove this reconciler and move its logic into ImageVersionReconciler.